app.rb

```diff
@@ -59,17 +60,11 @@ def create_a_student
     puts "\n"
   end
 
-  def create_a_teacher
+  def create_a_teacher(name, age)
     type = 'Teacher'
     puts 'Specialization: '
     specialization = gets.chomp
```

Comment:
- [ ] Great done on getting rid of the redundant code 👏. Please make sure that your methods in `app.rb,` such as `create person,` `create student,` and `create teacher,` as he pointed out, are only doing one thing, though, per the prior reviewer's request. Hence, in order to accomplish that, you should ensure that anything that receives user inputs is placed in `main.rb` or inside other classes or methods. 👌

### Hint: :point_down: 
Every `gets.chomp` should be moved


- [ **OPTIONAL**]  Please do the same thing with the other remaining methods like: `create_a_book`, `create_a_rental`, and `rental_person_id`. 😊

Reply:
@ridwanediallo Thanks for the hint! It helped understanding